specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview data Policies Service REST API Document",
+    "version": "2021-01-01-preview"
+  },
+  "paths": {
+    "/dataPolicies/{policyName}/scopes/{dataSource}": {
+      "delete": {
+        "tags": [
+          "DataPolicyBinding"
+        ],
+        "description": "Deletes a data policy binding",
+        "operationId": "DataPolicyBinding_Delete",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "policyName",
+            "description": "Name of the policy.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "dataSource",
+            "description": "Name of the datasource associated with the binding to be deleted.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
204 for no content

Reply:
Updated instances of 200 with empty body to 204